.github/ISSUE_TEMPLATE/web_api_type_definition_issue.yml

```diff
@@ -55,7 +55,7 @@ body:
     id: use_latest_typescript_web
     attributes:
       label: "Use the latest @types/web Package"
```

Comment:
No "use" but "have tried". Same for the label, options, and the TS version above.